controllers/admin/AdminQualityAssuranceController.php

```diff
@@ -197,6 +204,14 @@ public function ajaxProcessGetHookCallLogs()
         $this->renderJson($grouped);
     }
 
+    public function ajaxProcessToggleJSEventListener()
+    {
+        $oldStatus = Configuration::get('PS_QA_MODULE_LISTEN_JS');
+        $result = Configuration::updateValue('PS_QA_MODULE_LISTEN_JS', (bool) $oldStatus);
```

Comment:
```suggestion
        $result = Configuration::updateValue('PS_QA_MODULE_LISTEN_JS', (bool) !$oldStatus);
```

I think you wanted to toggle value? 🤔